chartfx-dataset/src/main/java/de/gsi/dataset/spi/DefaultAxisDescription.java

```diff
@@ -117,6 +118,15 @@ public boolean clear() {
 
     @Override
     public boolean equals(final Object obj) {
+        return equals(obj, 1e-6); // always fuzzy comparisons
+    }
+
+    /**
+     * @param obj object to compare to
+     * @param epsilon allow for fuzzy comparisons of min/max
+     * @return true: matches, false otherwise
+     */
+    public boolean equals(final Object obj, final double epsilon) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'equals(Object, double)' has an NPath complexity of 10368](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4728036)